ui/static/ui/js/learning_resources.jsx

```diff
@@ -88,6 +88,9 @@ define('learning_resources', [
                className="btn btn-white"
                onClick={this.selectXml}>Select XML</a>
           </p>
+          <p className="text-right">
+            <a href={this.state.previewUrl} target="_blank">Preview</a>
```

Comment:
consider making this a button instead of just a textual link with class `.btn-primary`.  I also think it would be nice to have it float to the left of the `Select XML` button instead of below it.